### PR TITLE
Update EDA examples csv data when creating records

### PR DESCRIPTION
**Pull request by umeditor**

### CODE REVIEW

**Comment by allisonletts** [comment]
@umeditor and @acrosman I'd love to move the fake data in here over to faker_edu now that it's a real thing. @umeditor can you list your source in the commit somewhere, or in a readme in the directory?

File: snowfakery_samples/EDA/Affiliation_EDA.macro.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+# Look at snowfakery_samples/salesforce/Account.recipe.yml for more examples.
+# - include_file: standard_EDA_objs_recipe.yml
+
+- plugin: snowfakery.standard_plugins.Salesforce.SalesforceQuery
+
+- macro: eda_affiliation_base
+  fields:
+    hed__Account__c:
+      SalesforceQuery.random_record:
```
[the remark this answers is not in the document]
Comment (umeditor) [reply snowfakery_samples/EDA/Affiliation_EDA.macro.yml:16]: Yes, this example was designed to be run against a new EDA org to populate multiple objects and not as piecemeal recipes.

File: snowfakery_samples/EDA/data/educational-institution.csv [hunk 2]
```diff
@@ -0,0 +1,201 @@
+name
+American University
```
[the remark this answers is not in the document]
Comment (umeditor) [reply snowfakery_samples/EDA/data/educational-institution.csv:2]: I'm going to use the new faker_edu provider to generate names.

**Comment by samanthashain** [comment]
@umeditor @allisonletts @acrosman it looks like this is very nearly ready to go.  do you need any support to complete the documentation on accessing EDU faker and approving this PR?

**Comment by acrosman** [comment]
The only hold up is that I haven't had a chance to validate the instructions and test the current recipe version.  I suspect we're good to go once we do those things.

**Review by acrosman — APPROVED** [review]

**Comment by acrosman** [comment]
Thanks for all the work that went into this.  It caused me to notice that our faker providers could use better instructions for how to install them to run with Snowfakery's pipx environment, but that's not a concern for this issue (or even this project).  Opened https://github.com/SFDO-Community-Sprints/Snowfakery-Edu/issues/19 and https://github.com/SFDO-Community-Sprints/Snowfakery-Nonprofit/issues/9 to resolve those.

**Pull request merged**